qa/kerberos/src/main/scala/org/opensearch/hadoop/qa/kerberos/spark/LoadToES.scala

```diff
@@ -53,8 +53,8 @@ class LoadToES(args: Array[String]) {
       .option("sep", "\t")
       .csv(args(0))
 
-    df.rdd.map(row => row.getValuesMap(row.schema.fieldNames)).saveToOpenSearch(s"${resource}_rdd")
-    df.saveToOpenSearch(s"${resource}_df")
+    // df.rdd.map(row => row.getValuesMap(row.schema.fieldNames)).saveToOpenSearch(s"${resource}_rdd")
```

Comment:
Why it is commented out?

Reply:
```
[Error] /local/home/hvamsi/clients/opensearch-hadoop/qa/kerberos/src/main/scala/org/opensearch/hadoop/qa/kerberos/spark/LoadToES.scala:56:64: value saveToOpenSearch is not a member of org.apache.spark.rdd.RDD[Map[String,Nothing]]
[Error] /local/home/hvamsi/clients/opensearch-hadoop/qa/kerberos/src/main/scala/org/opensearch/hadoop/qa/kerberos/spark/LoadToES.scala:57:8: value saveToOpenSearch is not a member of org.apache.spark.sql.DataFrame
[Error] /local/home/hvamsi/clients/opensearch-hadoop/qa/kerberos/src/main/scala/org/opensearch/hadoop/qa/kerberos/spark/ReadFromES.scala:51:24: value esJsonRDD is not a member of org.apache.spark.SparkContext
[Error] /local/home/hvamsi/clients/opensearch-hadoop/qa/kerberos/src/main/scala/org/opensearch/hadoop/qa/kerberos/spark/ReadFromES.scala:53:22: value esDF is not a member of org.apache.spark.sql.SQLContext
```

These are the build errors I get, not very sure what is going on here.

Reply:
:+1: I will take a look

Reply:
@harshavamsi please add these imports:

```
import org.opensearch.spark._
import org.opensearch.spark.sql._
```

and uncomment the test code